test_1:
  planning_all:
    time:

  planning_1:
    time:
    resources:
      cpu: [move_group]
      mem: [move_group]

  planning_2:
    time:
    resources:
      cpu: [grasping_app, move_group]
      mem: [grasping_app, move_group]
      io: [move_group]
      network: [move_group]

  planning_3:
    time:

  execution_all:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]

  execution_1:
    time:
    resources:
      cpu: [roscore]
      mem: [move_group]

  execution_2:
    time:
    path_length:
      - [base_link, gripper_right_grasp_link]
      - [base_link, arm_right_3_link]

  execution_3:
    time:
    resources:
      cpu: [grasping_app]
      mem: [roscore]

test_2:
  planning_1:
    time:

  execution_1:
    time: True
    resources:
      cpu: [move_group]
      mem: [move_group]
      io: [move_group]
      network: [move_group]
    obstacle_distance:
      - [arm_7, table]
      - [arm_7, arm_left_1, arm_left_7]
